src/main/resources/application.conf

```diff
@@ -536,6 +536,17 @@ mantis {
     # Iff true, any errors during metrics client operations will be logged.
     log-errors = true
   }
+
+  logging {
```

Comment:
Shouldn't we expose this config in the universal folder also? Maybe the file `logging.conf` will be useful.